walkthrough/garden_cookiecutter/{{cookiecutter.directory_name}}/{{cookiecutter.short_name}}.countries_exclude.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
Maybe instead of `*.countries_exclude.json`, the file could be called `*.excluded_countries.json`.